lib/opensusebasetest.pm

```diff
@@ -394,6 +394,22 @@ sub wait_boot {
     $self->{in_wait_boot} = 0;
 }
 
+sub enter_test_text {
```

Comment:
we have utils.pm and we have opensusebasetest.pm , do we have clear rule when we should move function to one and when to another ?